Dockerfile

```diff
@@ -178,6 +178,12 @@ COPY --chown=${USER} wait-for-it.sh manage.py backend_entrypoint.sh ${HOME}/
 COPY --chown=${USER} utils/ ${HOME}/utils
 COPY --chown=${USER} cvat/ ${HOME}/cvat
 
+ARG COVERAGE_PROCESS_START
+RUN if [ "${COVERAGE_PROCESS_START}" ]; then \
+        python3 -m  pip install --no-cache-dir coverage==7.2.3; \
```

Comment:
Why do we need the line? It should be installed by 75 line in the file.

Reply:
Updated